teams/codebender.md

```diff
@@ -9,15 +9,51 @@ permalink: /teams/codebender/
  - Thng Kai Yuan ([@thngkaiyuan](http://cs3281-2.blogspot.sg/))
    - Technology Expertise: Information Security
  - Moon Seokmin ([@bsmmoon](http://moonmoon3281.blogspot.sg/))
-   - Technology Expertise:
+   - Technology Expertise: Web Crawling
```

Comment:
Mostly web scraping. But I am also interested in other usage such as internet bot as well.